src/test/java/org/openelisglobal/dictionary/rest/controller/DictionaryMenuRestControllerTest.java

```diff
@@ -32,8 +34,9 @@ public class DictionaryMenuRestControllerTest extends BaseWebContextSensitiveTes
 
     @Before
     @Override
-    public void setUp() {
+    public void setUp() throws Exception {
         super.setUp();
+        executeDataSetWithStateManagement("testdata/dictionary.xml");
```

Comment:
@mherman22 , just confirming , does this test run fine if you dont load this dataset ??